cobra/cmd/licenses.go

```diff
@@ -425,6 +425,305 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 `,
 	}
 
+	Licenses["gpl2"] = License{
+		Name:            "GNU General Public License 3.0",
```

Comment:
Oops I missed this in the bit that I pasted. I will fix that in a moment.
On Feb 22, 2016 9:24 AM, "Eric Paris" <notifications@github.com> wrote:

> In cobra/cmd/licenses.go
> <https://github.com/spf13/cobra/pull/246#discussion_r53630675>:
>
> > @@ -425,6 +425,305 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
> >  `,
> >  	}
> >
> > +	Licenses["gpl2"] = License{
> > +		Name:            "GNU General Public License 3.0",
>
> 3.0 ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spf13/cobra/pull/246/files#r53630675>.
>